### ISSUE TITLE
UnusedLocalVariable false positive for $http_response_header

### ISSUE THREAD

**Issue by schinkel** [phpmd/phpmd#401]
Whenever `file_get_contents` is being called [`$http_response_header`](http://php.net/manual/en/reserved.variables.httpresponseheader.php) will be available in the local scope. 

This is falsely seen as UnusedLocalVariable violation in the following example code:

``` php
<?php
$url = '[url]';
file_get_contents($url);
foreach ($http_response_header as $header) {
    // do something
}
```

I use PHPMD 2.4.3.

**Comment by schinkel** [comment]
Thank you 👍
